### PR TITLE
#203 Redesign General setting

### PR DESCRIPTION
**Pull request by EugeneValko**
Screenshot_4
Screenshot_5
Screenshot_6

### CODE REVIEW

File: frontend/src/styles/material.sass [hunk 1]
```diff
@@ -79,12 +80,12 @@ snack-bar-container
 .mat-form-field-wrapper
     padding-bottom: 5px
 
-.mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle
-    border-color: $royal-blue
+::ng-deep .mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle
+    border-color: $midnight
```
Comment (YuraRiabov) [inline frontend/src/styles/material.sass:83]: I believe we don't need ng-deep when redefining styles in material.sass

File: frontend/src/styles/components/angular-material/mat-select.sass [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿::ng-deep mat-select
+    align-items: center !important
+    display: flex !important
+
+.mat-select
+    font-size: 14px
+    padding-left: 5px
+    border: 2px solid $pattens-blue
+    border-radius: 8px
+    height: 36px
```
Comment (YuraRiabov) [inline frontend/src/styles/components/angular-material/mat-select.sass]: I think it's not a good idea to have padding, border etc. set globally, please use ng-deep with custom classes in your component

**Review by YuraRiabov — CHANGES_REQUESTED** [review]

File: frontend/src/styles/material.sass (hunk 1 above)
Comment (EugeneValko) [reply frontend/src/styles/material.sass:83]: fix

File: frontend/src/styles/components/angular-material/mat-select.sass (hunk 2 above)
Comment (EugeneValko) [reply frontend/src/styles/components/angular-material/mat-select.sass]: fix

**Review by YuraRiabov — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ erDiagram
     int MaxNumberOfBookings
     int PaddingMeeting
     int MinBookingMeetingDifference
-    int Color
+	int Frequency
```
Comment (apofis96) [inline README.md]: use spaces instead of tab

**Review by apofis96 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (EvgenValko) [reply README.md]: Fix

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**